editor/src/clj/editor/gui.clj

```diff
@@ -3308,16 +3258,18 @@
   (input samplers [g/KeywordMap])
   (output samplers [g/KeywordMap] (gu/passthrough samplers))
 
-  (output own-gui-resource-type-names GuiResourceTypeNames :cached
-          (g/fnk [own-font-names layer-names own-material-infos own-particlefx-resource-names own-spine-scene-names own-texture-infos]
-            (let [own-material-names (coll/sorted-key-set own-material-infos)
-                  own-texture-names (coll/sorted-key-set own-texture-infos)]
-              {:font own-font-names
-               :layer layer-names
-               :material own-material-names
-               :particlefx own-particlefx-resource-names
-               :spine-scene own-spine-scene-names
-               :texture own-texture-names})))
+  (output aux-gui-resource-type-names GuiResourceTypeNames :cached
+          (g/fnk [aux-basic-gui-scene-info]
+            (let [material-names
+                  (coll/transfer (:material-infos aux-basic-gui-scene-info) (sorted-set)
+                    (map key)
+                    (remove coll/empty?))]
+              {:font (:font-names aux-basic-gui-scene-info)
+               :layer (:layer-names aux-basic-gui-scene-info)
+               :material material-names
+               :particlefx (:particlefx-resource-names aux-basic-gui-scene-info)
+               :spine-scene (:spine-scene-names aux-basic-gui-scene-info)
+               :texture (:texture-resource-names aux-basic-gui-scene-info)})))
```

Comment:
Used when renaming gui resources to make sure we don't rename anything in referencing gui scenes that is redeclared (or "shadowed") in the referencing scene.